ledger-tool/src/main.rs

```diff
@@ -1773,6 +1724,13 @@ fn main() {
                     let is_minimized = arg_matches.is_present("minimized");
                     let output_directory = value_t!(arg_matches, "output_directory", PathBuf)
                         .unwrap_or_else(|_| {
+                            let snapshot_archive_path = value_t!(matches, "snapshots", String)
+                                .ok()
+                                .map(PathBuf::from);
+                            let incremental_snapshot_archive_path =
+                                value_t!(matches, "incremental_snapshot_archive_path", String)
+                                    .ok()
+                                    .map(PathBuf::from);
```

Comment:
This is technically duplicate parsing. However, this PR knocked out duplicate parsing elsewhere, and allows a subsequent PR of mine to go forth without introducing more duplicate parsing. So, it makes things no worse and will enable things to be better in a PR that will consume this.

Longer term, I'd like to separate the parsing from `load_and_process_ledger()` into a different function, and then pass the parsed config in. This would allow us to get to single parse, but for now, I think this is good